### PR TITLE
tests: unit tests for DoJSON conferences

### PR DESCRIPTION
**Pull request by jacquerie**
Continues to address #1240.

- [ ] Conferences
    - [x] Test `acronym`
    - [x] Test `keywords`
    - [x] Test `nonpublic_note`
    - [x] Test `note`
    - [x] Test `series`
    - [x] Test `short_description`
    - [x] ~~Test `extra_place_info`~~ (should be merged with `contact_details`)

### CODE REVIEW

**Comment by jacquerie** [comment]
Ready for review.

**Comment by jacquerie** [comment]
@kaplun, care to take a look?

File: inspirehep/dojson/conferences/fields/bd1xx.py [hunk 1]
```diff
@@ -81,6 +81,12 @@ def alternative_titles(self, key, value):
 @utils.for_each_value
 def contact_details(self, key, value):
     """Contact details."""
+    extra_place_info = value.get('b')
+    if extra_place_info:
+        address = parse_conference_address(extra_place_info)
+        self.setdefault('address', [])
+        self['address'].append(address)
```
Comment (kaplun) [inline inspirehep/dojson/conferences/fields/bd1xx.py]: Can't you do it in one command? 

```python
self.setdefault('address', []).append(address)
```

**Comment by kaplun** [comment]
agh :) I hadn't finished review :)

**Comment by jacquerie** [comment]
Oh noes! But it was blocking the queue...

**Pull request merged**